bcda/cli.go

```diff
@@ -67,18 +120,28 @@ func importCCLF8(fileMetadata cclfFileMetadata) error {
 	)
 
 	sc := bufio.NewScanner(file)
+	count := 0
+	validator := cclfFileValidator["CCLF8"]
```

Comment:
I think it is possible for cclfFileValidator to be nil/empty (this might happen if CCLF0 file contents doesn't contain an entry for CCLF8 or CCLF9, or if the file is empty).  We should probably ensure that that the validator object contains the data we want.

Reply:
This won't even execute if the there's an error in reading the CCLF0 file. The ingestion of cclf0 happens first and if there's an error we log it and return. Because the ingestion of 8 and 9 has to be validated first, it doesn't make sense to continue if we have an empty cclf0 file or missing data points. But this is part of a larger discussion we had about what to do if cclf8 was weird and cclf9 was okay and vice versa.

Reply:
Seconding @msnook's comment. If the CCLF0 file is empty,`sc.Scan()` won't even be loop-able, right? So `importCCLF0()` could return an unpopulated `validator` map and no error.

Reply:
ahh yeah I see what you'll are saying, I was thinking more of a ingest error not an empty map. good stuff